tests/integration/control/test_control_resources.py

```
@@ -0,0 +1,960 @@
+
+from app.models.attack_resource_file import AttackResourceFile, AttackResourceType
+from app.models.project import ProjectUserAssociation, ProjectUserRole
+from app.models.user import User
+from tests.factories.project_factory import ProjectFactory
+
+
+async def create_resource(
+    db_session: AsyncSession,
+    project_id: int | None = None,
+    resource_type: AttackResourceType = AttackResourceType.WORD_LIST,
+    is_uploaded: bool = True,
```

Ruff (0.15.0)

[error] Boolean-typed positional argument in function definition

(FBT001)